front/app/containers/Admin/reporting/components/ReportBuilder/Templates/PlatformTemplate/index.tsx

```diff
@@ -20,28 +20,41 @@ import WhiteSpace from 'components/admin/ContentBuilder/Widgets/WhiteSpace';
 
 import { MessageDescriptor, useFormatMessageWithLocale } from 'utils/cl-intl';
```

Comment:
If you'd like to see it, [here](https://tan5841.epic.govocal.com/en/admin/reporting/report-builder/821da7b4-2370-492e-a347-e7f45f5afba5/editor) is the new Platform Template on epic.